Installed/Driver/Task/NonTS/Makefile

```diff
@@ -1,64 +1,34 @@
-##############################################################################

```

Comment:
Is it necessary to replace this by a generated file?

Reply:
The original NonTS Makefile was also automatically generated from the original mkmf by the look of things. It's probably okay for it to have been regenerated with the new mkmf, especially as it is consistent with the other task driver makefiles.

Reply:
There probably should be a Driver/Task/NonTS/local.mk file though, for the `PROTOCONST` and `LIBNAME` values which no longer exist in this file.